cmd/libs/go2idl/parser/parse_test.go

```diff
@@ -119,8 +127,12 @@ type FooObject interface {
 	CommonObject
 }
 
+
+func proto.AFunc( proto.Object proto.Object ) proto.Frobber {}
```

Comment:
"func proto.AFunc" doesn't make any sense to me at all? Maybe change to "var importedFunc = proto.AFunc" if you want to test like this. (Additionally there has to be a comma between parameters.)